packlink-build.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
suggestion: https://github.com/koalaman/shellcheck ouput:

```
In script.sh line 10:
cd packlink
^-- SC2164: Use cd ... || exit in case cd fails.


In script.sh line 20:
echo $1 > release.version
     ^-- SC2086: Double quote to prevent globbing and word splitting.


In script.sh line 23:
cd ..
^-- SC2103: Use a ( subshell ) to avoid having to cd back.

Reply:
shellcheck run